parts/k8s/kubernetesmastervars.t

```diff
@@ -1,6 +1,25 @@
     "etcdDiskSizeGB": "[parameters('etcdDiskSizeGB')]",
     "etcdDownloadURLBase": "[parameters('etcdDownloadURLBase')]",
     "etcdVersion": "[parameters('etcdVersion')]",
+    "etcdClientCertFilepath": "/etc/kubernetes/certs/etcd-client.crt",
+    "etcdClientKeyFilepath": "/etc/kubernetes/certs/etcd-client.key",
+    "etcdServerCertFilepath": "/etc/kubernetes/certs/etcd-server.crt",
+    "etcdServerKeyFilepath": "/etc/kubernetes/certs/etcd-server.key",
+    "etcdPeerCertFilepath":[
```

Comment:
Why is this part and below hard coded to max 5 peer?

Reply:
By definition, there can only be 1, 3 or 5 master profiles in a cluster (https://github.com/Azure/acs-engine/blob/master/docs/clusterdefinition.md#masterprofile-1)

Reply:
Got it. Thanks!